components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -405,14 +421,12 @@ - (void)layoutSubviews {
 }
 
 - (CGSize)sizeThatFits:(CGSize)size {
-  CGSize intrinsicContentSize = [self intrinsicContentSize];
-  return CGSizeMake(size.width, intrinsicContentSize.height);
-}
-
-- (CGSize)intrinsicContentSize {
-
-  CGFloat height = ([self usePadInsets] ? kNavigationBarPadDefaultHeight : kNavigationBarDefaultHeight);
-  return CGSizeMake(UIViewNoIntrinsicMetric, height);
+  CGFloat maxHeight =
+      [self usePadInsets] ? kNavigationBarPadDefaultHeight : kNavigationBarDefaultHeight;
+  CGFloat minHeight = kNavigationBarMinHeight;
+  CGFloat height =
+      self.useFlexibleTopBottomInsets ? MIN(MAX(size.height, minHeight), maxHeight) : maxHeight;
+  return CGSizeMake(size.width, height);
 }
```

Comment:
I've been under the impression that we had to implement intrinsicContentSize in order to support auto layout. Will removing this API break autolayout for clients?

Reply:
I'm not sure if it really breaks autolayout, and I can't find details on why it was added in https://github.com/material-components/material-components-ios/commit/c5cf080b3e2fc5cfd40357901d39f3ea3db04beb

I'll keep it here just to be safe.